app/assets/javascripts/Components/Modals/graders_distribution_modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Modal from "react-modal";
+import PropTypes from "prop-types";
+
+export class GraderDistributionModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      override: this.props.override,
```

Comment:
I don't think this state is necessary (or at least it doesn't seem to be used anywhere). If this is the case you can remove it, and in fact the overridden constructor altogether, since there's no other state.